packages/exhibitor-migration-script/extra/exhibitor-migration-script/bin/dcos-exhibitor-migrate-perform

```diff
@@ -0,0 +1,86 @@
+#!/usr/bin/env python3
+import copy
```

Comment:
this import order is off. Need to update `tox.ini` so that flake8 will scan / error on this file not matching our python style requirements.